### PR TITLE
intel-gtpin: new package

### PR DESCRIPTION
**Pull request by mwkrentel**
ping @rashawnlk and added her as maintainer with her permission

### CODE REVIEW

**Comment by mwkrentel** [comment]
ping @rashawnLK would you like to comment, or say, "I approve."
ping @tldahlgren or @adamjstewart would someone like to review and commit,
or else assign to someone else.

**Review by rashawnLK — DISMISSED** [review]
I approve

File: var/spack/repos/builtin/packages/intel-gtpin/package.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/intel-gtpin/package.py]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed the sha256.  Only request is to change the copyright date.

File: var/spack/repos/builtin/packages/intel-gtpin/package.py (hunk 1 above)
Comment (mwkrentel) [reply var/spack/repos/builtin/packages/intel-gtpin/package.py]: I updated the copyright, sorry I missed that.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/intel-gtpin/package.py]: Np. Thanks.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**